src/pages/registration/new_registration/LinkRegistrationForm.tsx

```diff
@@ -16,25 +16,39 @@ const StyledTextField = styled(TextField)`
   margin: 0 1rem 0 0;
 `;
 
-export interface DoiFormValues {
+interface DoiFormValues {
   doiUrl: string;
 }
 
 const emptyDoiFormValues: DoiFormValues = {
   doiUrl: '',
 };
 
-const doiUrlPlaceholder = 'https://doi.org/10.1000/xyz123';
+const doiUrlBase = 'https://doi.org/';
+const doiUrlPlaceholder = `${doiUrlBase}10.1000/xyz123`;
+const doiRegExp = new RegExp('\\b(10[.][0-9]{4,}(?:[.][0-9]+)*/(?:(?!["&\'<>])\\S)+)\\b'); // https://stackoverflow.com/a/10324802
```

Comment:
Ikke så verst? 😂 ser jo fryktelig ut. Men dette er uansett kun midlertidig da? Vi må ta bort denne funksjonaliteten i fremtiden når man skal kunne lime inn en lenke fra nyhetsartikkel f.eks? Er det noe vi vil gjøre egentlig? Vil gjerne ha en skriftlig beslutning der det står at vi skal tillate brukere å lime inn en ufullstendig lenke

Reply:
Hahah, tenkte mest på brukeropplevelsen når jeg skrev at det ikke ble så verst 😂 

kortversjon for hvordan det virker nå:
hvis brukeren har lagt inn en gyldig URL -> bruk den (uansett om den går til doi.org eller osteloff.party)
hvis ikke gyldig URL -> sjekk om det er en gyldig DOI i feltet, hent ut denne, og prefiks den med `https://doi.org/`

så man kan bruke en URL til hva som helst, men hvis man har en DOI (som ikke er URL) prøver vi vårt beste. Brukeren ser først fullstendig URL etter man trykker "SØK"

Reply:
Funksjonalitet kommer først og fremst fra kommentarer her: https://unit.atlassian.net/browse/NP-2036
Men tenker jeg kan ta en demo med JEG før vi evt merger

Reply:
ok, skjønner. ja, sikkert lurt med demo 👌

Reply:
Hadde en demo nå, og det så greit ut 👍 
Så lenge brukeren oppgir en URL er det jo greit, men så prøver vi å hjelpe litt hvis man bare oppgir en DOI også